custom/icds_reports/tests/reports/test_service_delivery.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import absolute_import
+
+from __future__ import unicode_literals
+from django.test import TestCase
+
+from custom.icds_reports.reports.service_delivery_dashboard import get_service_delivery_data
+
+
+class TestServiceDelivery(TestCase):
+
+    def test_get_service_delivery_data(self):
```

Comment:
These seem to have very similar filters. Can we add some tests for other filters (like the new age one)

Reply:
Added.